### PR TITLE
Creating a file for Sutton

### LINKED ISSUE #23: Create DaveHasteSpringer hometown file
## Sutton, UK
- [x] Create a branch
- [x] Create a file
- [x] Add a commit
- [x] Create a PR
- [x] Have a conversation
- [x] Get a 👍 
- [x] 🚢 it (merge)

### PR DESCRIPTION
**Pull request by DaveHasteSpringer**
This PR resolves #23

### CODE REVIEW

File: DaveHasteSpringer-sutton.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Sutton, UK
+
+
+### Places to eat
```
Comment (peppapig) [inline DaveHasteSpringer-sutton.md:4]: @DaveHasteSpringer I'd like to have some recommendations about yummy food.

File: DaveHasteSpringer-sutton.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
```
Comment (githubteacher) [inline DaveHasteSpringer-sutton.md:5]: @DaveHasteSpringer I love trying the favorite spots for locals - any recommendations?

Comment (DaveHasteSpringer) [inline DaveHasteSpringer-sutton.md:5]: Not really. The town is a dump. ;-)

**Comment by DaveHasteSpringer** [comment]
Yeah yeah, whatever.

**Pull request merged**